scripts/types/fixTypes.ts

```diff
@@ -1,32 +1,91 @@
 /**
- * -------------------------------------------------------
+ * -----------------------------------------------------------------
+ * The file is used to fix various issues with the types
+ * - Add reference paths to the mixins
+ * - Replace the dist triple slash reference with the webgpu types
+ * - Copy over the Shaders.d.ts file to the lib folder for
+ *   proper frag,vert,wgsl support
+ * -----------------------------------------------------------------
+ */
+
+import fs from 'fs';
+import glob from 'glob';
+import path from 'path';
```

Comment:
Minor future-proof import suggestions.

```suggestion
import * as fs from 'node:fs';
import * as glob from 'glob';
import * as path from 'node:path';
```